CHANGELOG.txt

```diff
@@ -20,6 +20,7 @@ Changes:
    * set origin evaluation status to "rejected" if nonlinloc reports the
      location run as "ABORTED", "IGNORED" or "REJECTED" (see #3230)
    * store "NLLOC" info header line in event and origin comments (see #3230)
+   * add reading of confidence ellipsoids for NLLoc V7 (see #3350)
```

Comment:
Looking at our test files, this was probably added during v6 life cycle at some point, there's a test file from 2015 that has this line. Maybe just leave away the "for NLLoc V7" part?